src/Altinn.Platform/Altinn.Platform.Storage/Storage/Wrappers/PartiesWrapper.cs

```diff
@@ -0,0 +1,31 @@
+using System.Net.Http;
+using System.Threading.Tasks;
+
+using Altinn.Platform.Storage.Clients;
+
+namespace Altinn.Platform.Storage.Wrappers
+{
+    /// <summary>
+    /// Represents a wrapper for a HttpClient targeting the SBL Bridge Register API endpoints.
```

Comment:
Is it the register or authz endpoint?

Reply:
Yes!
I'm actually not sure. I would love suggestions for naming here. SblBridge is using the PartiesController, but the endpoint is `authorization/api/partieswithinstances`. The data is stored in the Authorization database in a Schema called Register. Maybe I should have considered the endpoint and not how thing works in SBL?

Reply:
I landed on a complete refactoring of the PartyClient to PartiesWithInstancesClient instead. See updated code please.